<launch>
	<node pkg="bonxai_ros" exec="bonxai_server_node" name="bonxai_server">
	
		<!-- fVoxelGrid Resolution -->
		<param name="resolution" value="0.02" />

		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" value="map" />

		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model.max_range" value="4.0" />

		<!-- data source to integrate (PointCloud2) -->
		<!-- <remap from="cloud_in" to="cloud" /> -->

		<!-- Publish Height colormap on Resulting VoxelGrid -->
		<param name="use_height_map" value="true" />

	</node>
</launch>
